models/update.sh

```diff
@@ -1,5 +1,8 @@
 #!bin/bash
 
+# update constant baseline model
+bash models/constant-baseline/update.sh
```

Comment:
As you just running a single script here I'd likely go with `Rscript models/constant-baseline/update.R` and save yourself the bash step.